source/Engine.cpp

```diff
@@ -646,13 +647,14 @@ void Engine::Step(bool isActive)
 			if(it->IsDestroyed())
 				continue;
 
-			bool isEnemy = it->GetGovernment()->IsEnemy();
-			if(isEnemy || it->IsYours() || it->GetPersonality().IsEscort())
-			{
-				double width = min(it->Width(), it->Height());
-				statuses.emplace_back(it->Position() - center, it->Shields(), it->Hull(),
-					min(it->Hull(), it->DisabledHull()), max(20., width * .5), isEnemy);
-			}
+			if(it == player.FlagshipPtr())
```

Comment:
`flagship` is already the flagship pointer.
```suggestion
			if(it == flagship)
```